maquetado/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="book-detail">
+    <p class="book-title">Titulo del Libro</p>
+    <p class="book-description book-genre">(género)</p>
+  </div>
+  <div class="book-detail">
+    <p class="book-titleDescription">Autor del libro:</p>
```

Comment:
This is a different type of title so I would write `book-detail-title` and remember to make it an H3 tag